### DISCUSSION TITLE
Which planet in our solar system has the shortest day?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#572] [Galaxy Trivia] [answered]
**Bonus:** Which planet has the longest day?

**Comment by peterbaccaro** [comment, marked as answer]
**Jupiter** has the shortest day in the solar system. One day on Jupiter takes only about 10 hours (the time it takes for Jupiter to rotate or spin around once), and Jupiter makes a complete orbit around the Sun (a year in Jovian time) in about 12 Earth years (4,333 Earth days).

**Venus** has the longest day of all the planets. A day on Venus lasts 243 Earth days.
